Lombiq.NodeJs.Extensions/CustomExecTasks/ExclusiveMutex.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Diagnostics;
+using System.Threading;
+
+namespace Lombiq.NodeJs.Extensions.CustomExecTasks;
+
+public class ExclusiveMutex
+    : ICustomMutex
+{
+    private readonly string _mutexName;
+    private readonly TimeSpan _timeout;
+
+    public int RetryIntervalMs { get; set; } = 100;
+    public int WaitTimeMs { get; set; } = 1000;
+
+    public ExclusiveMutex(string mutexName, TimeSpan timeout)
+    {
+        _mutexName = mutexName;
+        _timeout = timeout;
+    }
+
+    public bool Execute(Func<bool> functionToExecute, Action<string, object[]> logWait, Action<string, object[]> logError)
```

Comment:
Can we use default values for `logWait` and `logError`?

```suggestion
    public bool Execute(Func<bool> functionToExecute, Action<string, object[]> logWait = null, Action<string, object[]> logError = null)
```